quinn-proto/src/connection.rs

```diff
@@ -2078,6 +2090,37 @@ impl Connection {
         }
     }
 
+    fn push_crypto(&mut self, crypto: Crypto, number: u64) {
```

Comment:
Could this be renamed to reflect that it's only used during 1RTT key updates? Perhaps rename the current `update_keys` to `initiate_key_update` and use `update_keys` for this?